php/Terminus/Models/Environment.php

```diff
@@ -779,4 +780,50 @@ private function elementAsDatabase($element) {
     return $element;
   }
 
+  /**
+   * Load site info
+   *
+   * @param [string] $key Set to retrieve a specific attribute as named
+   * @return [array] $info
+   */
+  public function info($key = null) {
+    $path = sprintf('environments/%s', $this->get('id'));
+    $result = \TerminusCommand::request(
+      'sites',
+      $this->site->get('id'),
+      $path,
+      'GET'
+    );
+    $info = array(
+      'id'                    => $this->get('id'),
+      'watchers'              => null,
+      'diffstat'              => null,
+      'on_server_development' => null,
+      'dns_zone'              => null,
+      'randseed'              => null,
+      'target_ref'            => null,
+      'lock'                  => null,
+      'target_commit'         => null,
+      'styx_cluster'          => null,
+    );
```

Comment:
Actually, Can we do the absolute minimum and then we'll add stuff as it's requested. So I think just 'id', 'on_server_development' (which we should probably coerce to Git/SFTP mode), and 'php_version'